### PR TITLE
[LG-5365] Update Production documentation

### PR DESCRIPTION
**Pull request by TiannaW24**
Update steps to create a production config: 
* Add step for prod config radio button
* Stress that "Prod" or "Production" should not be included in the app name or friendly name

### CODE REVIEW

File: _pages/production.md [hunk 1]
```diff
@@ -54,8 +54,9 @@ If you wish to deploy an application to our staging environment, then create a 
 
 Before you can deploy your application to the production environment, you will need to create a separate app on our dashboard that contains your production certificate, urls and logo. Here are the steps to complete your production configuration app:
 1. Create a new app on the dashboard <https://dashboard.int.identitysandbox.gov/>
-2. Enter the name of the application **as it appears in your IAA** in the "App Name" field
-3. Enter the name of the application **as you want it to appear to users** in the "Friendly Name" field
+2. Indicate that the app configuration is for a Production application 
+2. Enter the name of the application **as it appears in your IAA** in the "App Name" field - Do **NOT** use environment names such as "Prod" or "Production"
+3. Enter the name of the application **as you want it to appear to users** in the "Friendly Name" field - Do **NOT** use environment names such as "Prod" or "Production"
```
Comment (jauschalley) [inline _pages/production.md:59]: Could you be more specific? 🤡

**Review by jauschalley — APPROVED** [review]

File: _pages/production.md [hunk 2]
```diff
@@ -54,8 +54,9 @@ If you wish to deploy an application to our staging environment, then create a 
 
 Before you can deploy your application to the production environment, you will need to create a separate app on our dashboard that contains your production certificate, urls and logo. Here are the steps to complete your production configuration app:
 1. Create a new app on the dashboard <https://dashboard.int.identitysandbox.gov/>
-2. Enter the name of the application **as it appears in your IAA** in the "App Name" field
-3. Enter the name of the application **as you want it to appear to users** in the "Friendly Name" field
+2. Indicate that the app configuration is for a Production application
```
Comment (orenyk) [inline _pages/production.md:57]: **nitpick** there are two step `2`'s here - I'm pretty sure Markdown will handle it but figured I'd note it

File: _pages/production.md (hunk 1 above)
Comment (orenyk) [reply _pages/production.md:59]: **question:** one thought - what if they _want_ the environment name included in what is presented to users?

**Review by orenyk — APPROVED** [review]
Looks good! Two small comments

**Pull request merged**